web/satellite/src/components/team/AddUserPopup.vue

```diff
@@ -20,7 +20,7 @@
                             class="add-user__form-container__inputs-group__item"
                             :key="index" >
                                 <input
-                                    placeholder="test@mail.test"
+                                    placeholder="example@storj.io"
```

Comment:
I would use a reserved domain, like `example.com`.
Reference: https://tools.ietf.org/html/rfc2606

Reply:
would email@example.com be ok?

Reply:
it looks good to me!